### PR TITLE
Refactored 1.18 release calendar

### PR DESCRIPTION
**Pull request by alejandrox1**
This format is more readable.

Signed-off-by: alejandrox1 <alarcj137@gmail.com>

### CODE REVIEW

File: releases/release-1.18/README.md [hunk 1]
```diff
@@ -27,46 +27,46 @@
 
 The 1.18 release cycle is proposed as follows:
 
-- **Monday, January 06**: Week 1 - Release cycle begins
-- **Tuesday, January 28**: Week 4 - [Enhancements Freeze]
-- **Thursday, March 05**: Week 9 - [Code Freeze]
-- **Monday, March 16**: Week 11 - Docs must be completed and reviewed
-- **Tuesday, March 24**: Week 12 - Kubernetes v1.18.0 released
+- **Monday, January 06, 202**: Week 1 - Release cycle begins
```
Comment (onlydole) [inline releases/release-1.18/README.md]:
```suggestion
**Monday, January 06, 2020**: Week 1 - Release cycle begins
```

Comment (onlydole) [reply releases/release-1.18/README.md]: One small nitpick, otherwise LGTM! 👍

**Comment by alejandrox1** [comment]
Thank you for the review @onlydole

**Comment by justaugustus** [comment]
WAYYYYYY more readable.
Thanks for this @alejandrox1!

/lgtm
/approve

**Comment by justaugustus** [comment]
also:
/honk
/pony party
/woof

**Pull request merged**